libs/langchain-community/package.json

```diff
@@ -82,6 +82,7 @@
     "@langchain/scripts": ">=0.1.0 <0.2.0",
     "@langchain/standard-tests": "0.0.0",
     "@layerup/layerup-security": "^1.5.12",
+    "@libsql/client": "^0.14.0",
```

Comment:
This also needs to be an optional peer dep:

https://js.langchain.com/docs/contributing/code/#adding-an-entrypoint

Reply:
Added